appium/tests/specs/live/composeEmail/SendEmailToRecipientWithExpiredAndRevokedPublicKeys.spec.ts

```diff
@@ -26,7 +26,11 @@ describe('COMPOSE EMAIL: ', () => {
 
     await MailFolderScreen.clickCreateEmail();
     await NewMessageScreen.composeEmail(expiredPublicKey, emailSubject, emailText);
-    await NewMessageScreen.checkFilledComposeEmailInfo([expiredPublicKey], emailSubject, emailText);
+    await NewMessageScreen.checkFilledComposeEmailInfo({
+      recipients: [expiredPublicKey],
+      subject: emailSubject,
+      message: emailText }
+    );
```

Comment:
Are you using vscode or some other editor? Typically, vscode would auto-format and it would end up with a `});` which is the common way to format this.

Reply:
I'm using WebStorm and I can see that eslint was not configured correctly in my side.